docs/content/docs/dev/configuration/overview.md

```diff
@@ -0,0 +1,219 @@
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Project Configuration
+
+Every Flink application depends on a set of Flink libraries. At a minimum, the application depends
+on the Flink APIs and, in addition, on certain connector libraries (i.e. Kafka, Cassandra).
+When running Flink applications (either in a distributed deployment or locally for testing),
+the [Flink runtime library](https://ossindex.sonatype.org/component/pkg:maven/org.apache.flink/flink-runtime@1.14.3) 
+must be available.
+
+The guides in this section will show you how to configure your projects via popular build tools
+([Maven]({{< ref "docs/dev/configuration/maven" >}}), [Gradle]({{< ref "docs/dev/configuration/gradle" >}}),
+add the necessary dependencies (i.e. [connectors and formats]({{< ref "docs/dev/configuration/connector" >}}), 
+[testing]({{< ref "docs/dev/configuration/testing" >}})), and cover some 
+[advanced]({{< ref "docs/dev/configuration/advanced" >}}) configuration topics. 
+
+## Getting started
+
+To get started working on your Flink application, use the following commands, scripts, and templates 
+to create a Flink project.  
+
+{{< tabs "creating project" >}}
+{{< tab "Maven" >}}
+
+You can create a project based on an [Archetype](https://maven.apache.org/guides/introduction/introduction-to-archetypes.html)
+with the Maven command below or use the provided quickstart bash script.
+
+### Maven command
+```bash
+$ mvn archetype:generate                \
+  -DarchetypeGroupId=org.apache.flink   \
+  -DarchetypeArtifactId=flink-quickstart-java \
+  -DarchetypeVersion={{< version >}}
+```
+This allows you to name your newly created project and will interactively ask you for the groupId,
+artifactId, and package name.
+
+### Quickstart script
+```bash
+$ curl https://flink.apache.org/q/quickstart.sh | bash -s {{< version >}}
+```
+
+{{< /tab >}}
+{{< tab "Gradle" >}}
+You can create a project with a Gradle build script or use the provided quickstart bash script.
+
+### Gradle build script
+
+To execute these build configuration scripts, run the `gradle` command in the directory with these scripts.
+
+**build.gradle**
+
+```gradle
+plugins {
+    id 'java'
+    id 'application'
+    // shadow plugin to produce fat JARs
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+// artifact properties
+group = 'org.myorg.quickstart'
+version = '0.1-SNAPSHOT'
+mainClassName = 'org.myorg.quickstart.StreamingJob'
```

Comment:
`myorg.org` points to an actively used domain. I would suggest to just remove it.